homeassistant/components/utility_meter/sensor.py

```diff
@@ -264,7 +264,9 @@ async def async_added_to_hass(self):
             self._state = Decimal(state.state)
             self._unit_of_measurement = state.attributes.get(ATTR_UNIT_OF_MEASUREMENT)
             self._last_period = state.attributes.get(ATTR_LAST_PERIOD)
-            self._last_reset = state.attributes.get(ATTR_LAST_RESET)
+            self._last_reset = dt_util.parse_datetime(
+                state.attributes.get(ATTR_LAST_RESET)
+            )
```

Comment:
Seems unrelated to the PR description

Reply:
Not sure what's going on... because this patch is already in the dev branch:

https://github.com/home-assistant/core/blob/dev/homeassistant/components/utility_meter/sensor.py#L277

(belongs to a previous PR)

Reply:
Rebase needed?

Reply:
I removed this commit